charts/sonarqube/templates/_helpers.tpl

```diff
@@ -34,14 +34,16 @@ Selector labels
 {{- define "sonarqube.selectorLabels" -}}
 app: {{ include "sonarqube.name" . }}
 release: {{ .Release.Name }}
+app.kubernetes.io/name: {{ .Release.Name }}
```

Comment:
Doesn't make sense to replicate this to `sonarqube-dce` as well?

Reply:
Hello, original author here, I'm open to adding the same changes to DCE. I didn't because i wasn't aware of the chart, my org only uses the non-DCE version.

Reply:
@davividal, we have already the label used in DCE :) So no need to ask @wyattfry for extra help on this ;)